testing/sliding-sync-integration-test/assets/docker-compose.yml

```diff
@@ -28,8 +28,7 @@ services:
       - ./data/db:/var/lib/postgresql/data
 
   sliding-sync-proxy:
-    image: ghcr.io/matrix-org/sliding-sync:v0.99.0-rc1
-    # image: ghcr.io/matrix-org/sliding-sync:latest
+    image: ghcr.io/matrix-org/sliding-sync:latest
     links:
```

Comment:
Using `latest` probably makes things less deterministic, did you commit this intentionally?

Reply:
Yes, because a new release may be lacking.

Reply:
Fixed.